src/stories/index.stories.js

```diff
@@ -178,6 +178,17 @@ const dynamicLayersTemplate = `
 </v-mapbox>
 `
 
+const clickableLayersTemplate = `
+<v-mapbox
+ :map-style="style"
+ access-token="pk.eyJ1IjoibWFwYm94IiwiYSI6ImNpejY4NXVycTA2emYycXBndHRqcmZ3N3gifQ.rJcFIG214AriISLbB6B5aw"
+ style="height: 300px;"
+ :center="center"
+>
+  <v-mapbox-layer v-for="layer in layers" :options="layer" :key="layer.id" :clickable="true" @click="onLayerClick" />
```

Comment:
if you have a raster layer, it cannot be clickable in this way?

Reply:
that's true, they are never clickable. will put it in stories so devs know about this limitation